samples/arm-none-eabi/reported.by

```diff
@@ -0,0 +1,310 @@
+reporter_name="Jasmin Jessich"
+reporter_url="http://jasmin.anw.at/"
+reporter_comment="NOTE: Please read samples/arm-none-eabi/reported.by prior to any built!"
+
+#
+# This configuration will build an arm-none-eabi compiler to be used for
+# embedded development. The original build script can be found here:
+#    https://launchpad.net/gcc-arm-embedded
+# To see the original build script, you have to download the installer:
+#    https://launchpad.net/gcc-arm-embedded/5.0/5-2015-q4-major
+# 
+# I tried to mimic the original configure options and environment settings with
+# crosstool-ng (CT_NG) as close as possible. I really analysed every option
+# which was not provided by CT-NG or which was added. The description in this
+# file explains each diversion from the original.
+# 
+# NOTE:   newlib-nano is currently not built!
+# 
+# The original uses the git/SVN versions of binutils-gdb, gcc and newlib. CT-NG
+# doesn't currently support using a git or SVN repository for the source
+# location. In fact this is not really necessary, because the official packages
+# contain most of the required code already.
+# Only armv8-m is not available in the official packages. This core is very new
+# and even not available on GCC trunk, so I guess we can wait for the next GCC
+# release to add it.
+#
+# If someone really want to experiment with the mentioned core variant, feel
+# free to use them as custom location in CT-NG.
```

Comment:
As you may have noticed, I have another pull request that tries to restore most samples to a passing build. As far as I understand, this sample will not build without setting up the sources checked out from SVN/Git as custom locations. I just tried your branch and it indeed failed at CC core pass-2 with:
````
[CFG  ]    Error: --with-multilib-list=armv6-m,armv7-m,armv7e-m,armv7-r not supported.
[CFG  ]    Makefile:4046: recipe for target 'configure-gcc' failed
````

Therefore, it will get reported by `ct-ng build-all` as a failure. I'd suggest you implement `CT_GetSVN` in `scripts/functions` (`CT_GetGit` is already available); add the required support in `config/*` (see `mingw.in` for an example) and use the required Git/SVN locations in crosstool.config, rather than requiring some manual steps.

Please, don't add more broken samples than we already have. When pushing larger changes that are not constrained to a single architecture or triplet, @bhundven asks to regression test on all available targets and it is quite annoying to deal with such breakages.